.travis.yml

```diff
@@ -1,5 +1,15 @@
 language: java
+addons:
+  firefox: "56.0b12"
+  apt:
+    packages:
+      - xvfb
 sudo: false
+before_install:
+  - sudo apt-get install dbus dbus-x11 
```

Comment:
please add a comment that firefox will pollute log with warnings/errors about dbus events